README.md

```diff
@@ -1,6 +1,6 @@
 # Server Pipe [![Build Status](https://travis-ci.com/UniversityofWarwick/js-serverpipe.svg?branch=master)](https://travis-ci.com/UniversityofWarwick/js-serverpipe)
 
-This is a basic wrapper for `cross-fetch` that we use to make authenticated requests back to the application server. This replaces
+This is a basic wrapper for `fetch()` that we use to make authenticated requests back to the application server. This replaces
```

Comment:
I don't think the second sentence makes sense any more here, it was added when we switched from isomorphic-fetch to cross-fetch but native fetch doesn't "replace" anything. I'd just drop the sentence entirely

Reply:
This is done